src/test/java/CarComfortServiceTest.java

```diff
@@ -0,0 +1,243 @@
+import exception.CarRentalException;
+import models.cars.CarComfort;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import repository.CarComfortRepository;
+import services.car.CarComfortService;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+public class CarComfortServiceTest {
+    private CarComfortRepository carComfortRepositoryMock;
+    private CarComfortService carComfortService;
+
+    @BeforeEach
+    public void setUp() {
+        carComfortRepositoryMock = mock(CarComfortRepository.class);
+        assertNotNull(carComfortRepositoryMock);
+
+        carComfortService = new CarComfortService(carComfortRepositoryMock);
+        assertNotNull(carComfortService);
+    }
+
+
+    @Test
+    public void whenInsertCorrectCarComfort_thenVerifyCarComfort() {
+        CarComfort carComfort = buildCarComfort(UUID.randomUUID());
+
+        when(carComfortRepositoryMock.insert(carComfort)).thenReturn(carComfort);
+
+        CarComfort response = carComfortService.insert(carComfort);
+        assertNotNull(response);
+
+        assertEquals(carComfort.getId(), response.getId());
```

Comment:
May be you can use here: assertEquals(carComfort, response)

Reply:
Done